config/config_params.go

```diff
@@ -166,16 +166,17 @@ type Config struct {
 	WireguardMTU                   int    `config:"int;0"`
 	WireguardHostEncryptionEnabled bool   `config:"bool;false"`
 
-	BPFEnabled                         bool           `config:"bool;false"`
-	BPFDisableUnprivileged             bool           `config:"bool;true"`
-	BPFLogLevel                        string         `config:"oneof(off,info,debug);off;non-zero"`
-	BPFDataIfacePattern                *regexp.Regexp `config:"regexp;^((en|wl|ww|sl|ib)[opsx].*|(eth|wlan|wwan).*|tunl0$|vxlan.calico$|wireguard.cali$)"`
-	BPFConnectTimeLoadBalancingEnabled bool           `config:"bool;true"`
-	BPFExternalServiceMode             string         `config:"oneof(tunnel,dsr);tunnel;non-zero"`
-	BPFKubeProxyIptablesCleanupEnabled bool           `config:"bool;true"`
-	BPFKubeProxyMinSyncPeriod          time.Duration  `config:"seconds;1"`
-	BPFKubeProxyEndpointSlicesEnabled  bool           `config:"bool;false"`
-	BPFExtToServiceConnmark            int            `config:"int;0"`
+	BPFEnabled                         bool             `config:"bool;false"`
+	BPFDisableUnprivileged             bool             `config:"bool;true"`
+	BPFLogLevel                        string           `config:"oneof(off,info,debug);off;non-zero"`
+	BPFDataIfacePattern                *regexp.Regexp   `config:"regexp;^((en|wl|ww|sl|ib)[opsx].*|(eth|wlan|wwan).*|tunl0$|vxlan.calico$|wireguard.cali$)"`
+	BPFConnectTimeLoadBalancingEnabled bool             `config:"bool;true"`
+	BPFExternalServiceMode             string           `config:"oneof(tunnel,dsr);tunnel;non-zero"`
+	BPFKubeProxyIptablesCleanupEnabled bool             `config:"bool;true"`
+	BPFKubeProxyMinSyncPeriod          time.Duration    `config:"seconds;1"`
+	BPFKubeProxyEndpointSlicesEnabled  bool             `config:"bool;false"`
+	BPFExtToServiceConnmark            int              `config:"int;0"`
+	BPFPSNATPorts                      numorstring.Port `config:"portrange;20000:29999"`
```

Comment:
API PR says "20999" but here you have "29999"

Reply:
I had this (29999) in mind.